Automattic/woocommerce-payments
# Change `Dispute → Details` links to `Transaction → Details`

Fixes #6929

> **Warning**
> This PR is **NOT** behind a feature flag

#### Changes proposed in this Pull Request
The goal of this PR is to update any links to disputes anywhere in the UI. As we are now using the transaction detail page for managing and surfacing disputes, links to "a dispute" should point to the relevant transaction detail URL. 

Here's a list of places in the UI that are affected (might not be complete, please explore!):

  - Payments overview reminder notice - `Respond to dispute for $33 now!`
  - Transaction detail timeline
    - `mapEvents` `dispute_needs_response` ?
  - Dispute list – row click, button click
  - Order service (PHP)
	  - Order note when dispute created
	  - Order note when dispute closed
  - `dispute_updated` webhook handler
	  - Transaction note when dispute funds withdrawn
	  - Transaction note when dispute funds reinstated
	  - Transaction note when dispute updated (unsure what causes this)
  - Order edit screen: disputed order notice

#### Testing instructions
Review the WooPayments admin UI for places that mention or might link to disputes and ensure they now link to the correct transaction detail screen. Ensure that the transaction screen shows the relevant info about the dispute and the merchant can action, resolve, respond to the dispute.

TBD - list of specific places this PR has fixed 

-------------------


**Post merge**
